### ISSUE TITLE
Data structure updated

### ISSUE THREAD

**Issue by usmanokara** [CSSEGISandData/COVID-19#1425]
Old files
time_series_19-covid-Confirmed.csv
time_series_19-covid-Deaths.csv
time_series_19-covid-Recovered.csv

New file

time_series_covid19_confirmed_global.csv
time_series_covid19_deaths_global.csv
time_series_covid19_recovered_global.csv is missing please provide this file also

**Comment by Hikmer** [comment]
Yes where is the recovered file?  Many people now have to rebuild dashboard and reports which is very inconvenient at a time like this.

**Comment by Alfacharlie1973** [comment]
time_series_covid19_recovered_global.csv will not be generated as the have communicated before. 
They were planning to add another view with testing progress but that has not been posted yet.
Check Issue #1250 .

**Comment by tghkwomack** [comment]
What about the state- and county-level data?  At what point will that be posted?

**Comment by Munsky** [comment]
Hi.  If possible, it would be nice to see the recovered data (e.g., "time_series_covid19_recovered_global.csv")  for other analyses of the data (e.g., the Matlab tool here: https://github.com/MunskyGroup/Matlab_Wrap_for_JHU_COVID19).  These could be useful to others for the building of models, even if there is no plan to include in the online visuals.

Testing data would also be very nice to include in such models.

P.S.  Thank you for the excellent tool and data sharing.  This is a great service to the community.
